en/middleware/micrortps.md

```diff
@@ -400,7 +400,7 @@ Enter the following commands to create the application:
 cd /path/to/PX4/Firmware/build/px4_sitl_rtps/src/modules/micrortps_bridge
 mkdir micrortps_listener
 cd micrortps_listener
-fastrtpsgen -example x64Linux2.6gcc ../micrortps_client/micrortps_agent/idl/sensor_combined.idl
+fastrtpsgen -example x64Linux2.6gcc ../micrortps_client/micrortps_agent/idl/sensor_combined_.idl
```

Comment:
This isn't correct. I just generated the agent using `make px4_sitl_rtps` and the files do not have trailing underscores:
```sh
├── idl
│   ├── adc_report.idl
│   ├── airspeed.idl
│   ├── battery_status.idl
│   ├── camera_capture.idl
│   ├── camera_trigger.idl
│   ├── collision_constraints.idl
│   ├── collision_report.idl
│   ├── cpuload.idl
│   ├── debug_array.idl
│   ├── debug_key_value.idl
│   ├── debug_value.idl
│   ├── debug_vect.idl
│   ├── distance_sensor.idl
│   ├── estimator_status.idl
│   ├── home_position.idl
│   ├── iridiumsbd_status.idl
│   ├── obstacle_distance.idl
│   ├── onboard_computer_status.idl
│   ├── optical_flow.idl
│   ├── position_setpoint.idl
│   ├── position_setpoint_triplet.idl
│   ├── radio_status.idl
│   ├── satellite_info.idl
│   ├── sensor_baro.idl
│   ├── sensor_combined.idl
│   ├── sensor_selection.idl
│   ├── timesync.idl
│   ├── trajectory_waypoint.idl
│   ├── vehicle_attitude.idl
│   ├── vehicle_mocap_odometry.idl
│   ├── vehicle_odometry.idl
│   ├── vehicle_trajectory_waypoint.idl
│   ├── vehicle_visual_odometry.idl
│   ├── vtol_vehicle_status.idl
│   └── wind_estimate.idl
```
You might have a problem with your FastRTPS installation where it is finding a version you have installed that is equal or lower than 1.7.2 - https://github.com/PX4/Firmware/blob/master/msg/tools/px_generate_uorb_topic_files.py#L200-L202. Please check on your system for the `libfastrps` links using `ldconfig -v | grep libfastrtps`, and make sure that you only have 1.8.0 or above.

Reply:
looks like I have 1.8.3 installed.  

/sbin/ldconfig.real: Can't stat /usr/local/lib/x86_64-linux-gnu: No such file or directory
/sbin/ldconfig.real: Path `/lib/x86_64-linux-gnu' given more than once
/sbin/ldconfig.real: Path `/usr/lib/x86_64-linux-gnu' given more than once
/sbin/ldconfig.real: /lib/x86_64-linux-gnu/ld-2.27.so is the dynamic linker, ignoring

	libfastrtps.so.1 -> libfastrtps.so.1.8.3
/sbin/ldconfig.real: /lib32/ld-2.27.so is the dynamic linker, ignoring

Reply:
What's the output of `fastrtpsgen -version`?

Reply:
fastrtpsgen -version
openjdk version "11.0.6" 2020-01-14
OpenJDK Runtime Environment (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.6+10-post-Ubuntu-1ubuntu118.04.1, mixed mode, sharing)
fastrtpsgen version 1.0.4

Reply:
@TSC21 Hey have you been able to see why with my versions it might not have the same files you are building?

Reply:
The generated IDL's terminate without `_`. So, not applicable.